lib/srv/server/installer/defaultinstallers.go

```diff
@@ -26,19 +26,50 @@ import (
 	"github.com/gravitational/teleport/lib/web/scripts/oneoff"
 )
 
-// DefaultInstaller represents the default installer script provided by teleport.
-var DefaultInstaller = oneoffScriptToDefaultInstaller()
+// Depending on your cluster setup we have 2 installers:
+//   - LegacyDefaultInstaller which uses oneoff.sh to download teleport and run "teleport install".
+//     Teleport install does package-based installations.
+//   -
```

Comment:
What's up with the formatting of this comment?

Reply:
Leftover comment that I forgot to remove